providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/storage/connector/sync/SyncStreamQueueSource.java

```diff
@@ -157,9 +157,9 @@ private void observeSyncStream() throws InterruptedException {
 
                     Throwable streamException = taken.getError();
                     if (streamException != null) {
-                        log.debug("Exception in GRPC connection, streamException {}, will reconnect", streamException);
-                        if (!outgoingQueue.offer(new QueuePayload(
-                                QueuePayloadType.ERROR, "Error from stream or metadata", metadataResponse))) {
+                        log.error("Exception in GRPC connection, streamException {}, will reconnect", streamException);
```

Comment:
I made this `debug` recently because it's seen when the stream deadline is exceeded (a somewhat expected occurrence). 

However, this also can hide other errors which can make troubleshooting hard, so I'm bringing it back. I think seeing stream deadlines every 10 minutes or whatever is acceptable when you've configured them.